src/SDKs/CognitiveServices/dataPlane/Vision/ComputerVision/ComputerVision/Generated/SdkInfo_ComputerVisionClient.cs

```diff
@@ -45,10 +48,10 @@ internal static partial class SdkInfo
       // BEGIN: Code Generation Metadata Section
       public static readonly String AutoRestVersion = "latest";
       public static readonly String AutoRestBootStrapperVersion = "autorest@2.0.4283";
-      public static readonly String AutoRestCmdExecuted = "cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/cognitiveservices/data-plane/ComputerVision/readme.md --csharp --version=latest --reflect-api-versions --csharp-sdks-folder=C:\\Users\\mamirzam\\CV-API\\azure-sdk-for-net\\src\\SDKs";
-      public static readonly String GithubForkName = "Azure";
-      public static readonly String GithubBranchName = "master";
-      public static readonly String GithubCommidId = "a2323a5bf8cd642bee954f7b49ff4fe1976cdeec";
+      public static readonly String AutoRestCmdExecuted = "cmd.exe /c autorest.cmd https://github.com/toothache/azure-rest-api-specs/blob/ocr_read/specification/cognitiveservices/data-plane/ComputerVision/readme.md --csharp --version=latest --reflect-api-versions --csharp-sdks-folder=E:\\Source\\cognition\\azure-sdk-for-net\\src\\SDKs";
+      public static readonly String GithubForkName = "toothache";
```

Comment:
Will generate SDK using code from master branch of `azure-rest-api-specs` once the [PR](https://github.com/Azure/azure-rest-api-specs/pull/5314) is approved and gets merged.

[Computer Vision] Add read to CV OCR API. #5314
https://github.com/Azure/azure-rest-api-specs/pull/5314